app-frontend/src/app/components/pages/project/layer/scenes/browse/index.html

```diff
@@ -1,138 +1,156 @@
-    Filter...
-  </button>
-  <!-- <select class="btn btn-small btn-transparent" -->
-  <!-- ng-model="$ctrl.ordering"> -->
-  <!-- <option value="">Newest</option> -->
-  <!-- <option value="manual">Manual</option> -->
-  <!-- <option value="date">Date</option> -->
-  <!-- </select> -->
-  <button class="btn btn-small btn-transparent"
-          ng-click="$ctrl.visibleScenes.size > 0 && $ctrl.hideAll()"
-          ng-disabled="$ctrl.visibleScenes.size === 0"
-          title="Hide all visible scene boundaries on map"
-  >
-    Hide all
-  </button>
+    <a class="btn btn-small btn-transparent" ui-sref="project.layer.scenes">
+        <i class="icon-caret-left"></i>Back
+    </a>
+    <div class="flex-fill"></div>
+    <button class="btn btn-small btn-transparent" ng-click="$ctrl.showFilterPane = true">
+        Filter...
+    </button>
+    <!-- <select class="btn btn-small btn-transparent" -->
+    <!-- ng-model="$ctrl.ordering"> -->
+    <!-- <option value="">Newest</option> -->
+    <!-- <option value="manual">Manual</option> -->
+    <!-- <option value="date">Date</option> -->
+    <!-- </select> -->
+    <button
+        class="btn btn-small btn-transparent"
+        ng-click="$ctrl.visibleScenes.size > 0 && $ctrl.hideAll()"
+        ng-disabled="$ctrl.visibleScenes.size === 0"
+        title="Hide all visible scene boundaries on map"
+    >
+        Hide all
+    </button>
 </div>
 <div class="selected-actions-group" ng-show="$ctrl.selected.size > 0">
-  <rf-selected-actions-bar
-      checked="$ctrl.allVisibleSelected()"
-      on-click="$ctrl.selectAll()"
-      action-text="$ctrl.selectText"
-  >
-    <button class="btn btn-transparent" ng-click="$ctrl.addScenesToLayer()">Add to layer</button>
-  </rf-selected-actions-bar>
+    <rf-selected-actions-bar
+        checked="$ctrl.allVisibleSelected()"
+        on-click="$ctrl.selectAll()"
+        action-text="$ctrl.selectText"
+    >
+        <button class="btn btn-transparent" ng-click="$ctrl.addScenesToLayer()"
+            >Add to layer</button
+        >
+    </rf-selected-actions-bar>
 </div>
 <div class="list-group" ng-if="!$ctrl.fetchingScenes">
-  <div class="list-group-item"
-       ng-if="$ctrl.currentRepository && !$ctrl.currentRepository.service.defaultRepository"
-  >
-    <div class="alert alert-default">
-      <div class="alert-message">Any scenes added from this repository will also appear in your imports</div>
+    <div
+        class="list-group-item"
+        ng-if="$ctrl.currentRepository && !$ctrl.currentRepository.service.defaultRepository"
+    >
+        <div class="alert alert-default">
+            <div class="alert-message"
+                >Any scenes added from this repository will also appear in your imports</div
+            >
+        </div>
+    </div>
+    <div class="list-group-item" ng-if="$ctrl.sceneCount !== '0'">
+        <strong class="color-dark">
+            1 - {{ $ctrl.sceneList.length }} of {{ $ctrl.sceneCount }} scenes
```

Comment:
For this `1 - ...` display, I think we could either do better checks on length or count in here to see if we want to display `1 -` or not. Or we could use the pagination count control instead.

Reply:
I'll address this in my next PR which will be another round of UI cleanup.